### ISSUE TITLE
Looks like MRO is unusable

### ISSUE THREAD

**Issue by meteozond** [stefanhoelzl/vue.py#44]
```python
from vue import *


class Parent(VueComponent):
    @computed
    def my_computed(self):
        return 'Parent'

    template = '<div>Hello {{my_computed}}!</div>'


class Child(Parent):
    @computed
    def my_computed(self):
        return f'Child of {super().my_computed()}'


Child('#app')
```

```
Javascript exception: TypeError: Cannot read properties of undefined (reading '__mro__')
    at $$super.__getattribute__ (vuepy.js:7406:73)
    at $B.$getattr (vuepy.js:6953:13)
    at my_computed1 (eval at run_py (vuepy.js:9261:14), <anonymous>:160:83)
    at Vue.wrapper63 (eval at run_py (vuepy.js:9261:14), <anonymous>:360:185)
    at Vue.<anonymous> (vuepy.js:9027:26)
    at Watcher.get (vue.js:4478:27)
    at Watcher.evaluate (vue.js:4583:23)
    at Proxy.computedGetter (vue.js:4832:19)
    at Proxy.eval (eval at createFunction (vue.js:11649:14), <anonymous>:3:44)
    at Vue._render (vue.js:3551:24)
vuepy.js:7875 Array(5)0: "app_Child_4_my_computed_5"1: {self: {…}, $line_info: '15,app', $f_trace: {…}, __annotations__: {…}, $parent: undefined, …}2: "app"3: {__class__: {…}, __name__: 'app', __doc__: {…}, __package__: '', __loader__: {…}, …}4: ƒ my_computed1(self)length: 5[[Prototype]]: Array(0)
vuepy.js:7908 test is js exc is recursion error TypeError: Cannot read properties of undefined (reading '__mro__')
    at $$super.__getattribute__ (vuepy.js:7406:73)
    at $B.$getattr (vuepy.js:6953:13)
    at my_computed1 (eval at run_py (vuepy.js:9261:14), <anonymous>:160:83)
    at Vue.wrapper63 (eval at run_py (vuepy.js:9261:14), <anonymous>:360:185)
    at Vue.<anonymous> (vuepy.js:9027:26)
    at Watcher.get (vue.js:4478:27)
    at Watcher.evaluate (vue.js:4583:23)
    at Proxy.computedGetter (vue.js:4832:19)
    at Proxy.eval (eval at createFunction (vue.js:11649:14), <anonymous>:3:44)
    at Vue._render (vue.js:3551:24) TypeError: Cannot read properties of undefined (reading '__mro__')
vuepy.js:7876 recursion error ? false
vuepy.js:7908 test is js exc is recursion error TypeError: Cannot read properties of undefined (reading '__mro__')
    at $$super.__getattribute__ (vuepy.js:7406:73)
    at $B.$getattr (vuepy.js:6953:13)
    at my_computed1 (eval at run_py (vuepy.js:9261:14), <anonymous>:160:83)
    at Vue.wrapper63 (eval at run_py (vuepy.js:9261:14), <anonymous>:360:185)
    at Vue.<anonymous> (vuepy.js:9027:26)
    at Watcher.get (vue.js:4478:27)
    at Watcher.evaluate (vue.js:4583:23)
    at Proxy.computedGetter (vue.js:4832:19)
    at Proxy.eval (eval at createFunction (vue.js:11649:14), <anonymous>:3:44)
    at Vue._render (vue.js:3551:24) TypeError: Cannot read properties of undefined (reading '__mro__')
vuepy.js:7874 Javascript exception: TypeError: Cannot read properties of undefined (reading '__mro__')
    at $$super.__getattribute__ (vuepy.js:7406:73)
    at $B.$getattr (vuepy.js:6953:13)
    at my_computed1 (eval at run_py (vuepy.js:9261:14), <anonymous>:160:83)
    at Vue.wrapper63 (eval at run_py (vuepy.js:9261:14), <anonymous>:360:185)
    at Vue.<anonymous> (vuepy.js:9027:26)
    at Watcher.get (vue.js:4478:27)
    at Watcher.evaluate (vue.js:4583:23)
    at Proxy.computedGetter (vue.js:4832:19)
    at Proxy.eval (eval at createFunction (vue.js:11649:14), <anonymous>:3:44)
    at Vue._render (vue.js:3551:24)
vuepy.js:7875 Array(5)
vuepy.js:7908 test is js exc is recursion error TypeError: Cannot read properties of undefined (reading '__mro__')
    at $$super.__getattribute__ (vuepy.js:7406:73)
    at $B.$getattr (vuepy.js:6953:13)
    at my_computed1 (eval at run_py (vuepy.js:9261:14), <anonymous>:160:83)
    at Vue.wrapper63 (eval at run_py (vuepy.js:9261:14), <anonymous>:360:185)
    at Vue.<anonymous> (vuepy.js:9027:26)
    at Watcher.get (vue.js:4478:27)
    at Watcher.evaluate (vue.js:4583:23)
    at Proxy.computedGetter (vue.js:4832:19)
    at Proxy.eval (eval at createFunction (vue.js:11649:14), <anonymous>:3:44)
    at Vue._render (vue.js:3551:24) TypeError: Cannot read properties of undefined (reading '__mro__')
vuepy.js:7876 recursion error ? false
vuepy.js:7908 test is js exc is recursion error TypeError: Cannot read properties of undefined (reading '__mro__')
    at $$super.__getattribute__ (vuepy.js:7406:73)
    at $B.$getattr (vuepy.js:6953:13)
    at my_computed1 (eval at run_py (vuepy.js:9261:14), <anonymous>:160:83)
    at Vue.wrapper63 (eval at run_py (vuepy.js:9261:14), <anonymous>:360:185)
    at Vue.<anonymous> (vuepy.js:9027:26)
    at Watcher.get (vue.js:4478:27)
    at Watcher.evaluate (vue.js:4583:23)
    at Proxy.computedGetter (vue.js:4832:19)
    at Proxy.eval (eval at createFunction (vue.js:11649:14), <anonymous>:3:44)
    at Vue._render (vue.js:3551:24) TypeError: Cannot read properties of undefined (reading '__mro__')
vuepy.js:9027 TypeError: Cannot read properties of undefined (reading '__mro__')
    at $$super.__getattribute__ (vuepy.js:7406:73)
    at $B.$getattr (vuepy.js:6953:13)
    at my_computed1 (eval at run_py (vuepy.js:9261:14), <anonymous>:160:83)
    at Vue.wrapper63 (eval at run_py (vuepy.js:9261:14), <anonymous>:360:185)
    at Vue.<anonymous> (vuepy.js:9027:26)
    at Watcher.get (vue.js:4478:27)
    at Watcher.evaluate (vue.js:4583:23)
    at Proxy.computedGetter (vue.js:4832:19)
    at Proxy.eval (eval at createFunction (vue.js:11649:14), <anonymous>:3:44)
    at Vue._render (vue.js:3551:24)
vue.js:634 [Vue warn]: Error in render: "Error"

(found in <Root>)
warn @ vue.js:634
vue.js:1897 Error
    at _b_.AttributeError.$factory (eval at $make_exc (vuepy.js:7929:5), <anonymous>:41:354)
    at $B.$getattr (vuepy.js:6884:26)
    at Object.getattr (vuepy.js:6848:11)
    at Vue.<anonymous> (vuepy.js:9028:17)
    at Watcher.get (vue.js:4478:27)
    at Watcher.evaluate (vue.js:4583:23)
    at Proxy.computedGetter (vue.js:4832:19)
    at Proxy.eval (eval at createFunction (vue.js:11649:14), <anonymous>:3:44)
    at Vue._render (vue.js:3551:24)
    at Vue.updateComponent (vue.js:4067:23)
logError @ vue.js:1897
```

**Comment by stefanhoelzl** [comment]
Thanks for reporting!

**TL;DR**
get the latest changes on master and make a slight modification to your code
```python
from vue import *

class Parent(VueComponent):
    @computed
    def my_computed(self):
        return 'Parent'

    template = '<div>Hello {{my_computed}}!</div>'

class Child(Parent):
    @computed
    def my_computed(self):
        value = super().my_computed()
        return f'Child of {value}'

Child('#app')
```

-----------------
There are actually two bugs when using `f'Child of {super().my_computed()}'`

1. the one you discovered with the `__mro__` one you discovered
This seems to be a bug in the brython version used by vue.py (3.8.9), when using `super()` within a f-string.
```python
class A:
    def foo(self):
       return "A"

class B(A):
    def foo(self):
        return f"{super().foo()}"
```
raises the same exception
but this works
```python
class A:
    def foo(self):
       return "A"

class B(A):
    def foo(self):
        value = super().foo()
        return f"{value}"
```
the latest brython version 3.11.1 seems to have fixed this. But there is some work required on vue.py to be able to bump the version.

2. computed properties where not callable in vue.py
for this issue I just push a fix to master

**Comment by meteozond** [comment]
Wow this is super!
